unity-renderer/Assets/Rendering/ProceduralSkybox/ToolProceduralSkybox/Scripts/Editor/Procedural Skybox/Layers3D/RenderSatellite3DLayer.cs

```diff
@@ -0,0 +1,51 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+
+namespace DCL.Skybox
+{
+    public static class RenderSatellite3DLayer
+    {
+        public static void RenderLayer(ref float timeOfTheDay, EditorToolMeasurements toolSize, Config3DSatellite config)
+        {
+
+            // name In Editor
+            EditorGUILayout.BeginHorizontal();
+            EditorGUILayout.LabelField(SkyboxEditorLiterals.layerName, GUILayout.Width(150), GUILayout.ExpandWidth(false));
+            config.nameInEditor = EditorGUILayout.TextField(config.nameInEditor, GUILayout.Width(200), GUILayout.ExpandWidth(false));
+            EditorGUILayout.EndHorizontal();
+
+            EditorGUILayout.Separator();
+
+            // Time Span
+            RenderSimpleValues.RenderSepratedFloatFields(SkyboxEditorLiterals.timeSpan, SkyboxEditorLiterals.start, ref config.timeSpan_start, SkyboxEditorLiterals.end, ref config.timeSpan_End);
+            SkyboxEditorUtils.ClampToDayTime(ref config.timeSpan_start);
+            SkyboxEditorUtils.ClampToDayTime(ref config.timeSpan_End);
+
+            // Fading
+            RenderSimpleValues.RenderSepratedFloatFields(SkyboxEditorLiterals.fade, SkyboxEditorLiterals.inStr, ref config.fadeInTime, SkyboxEditorLiterals.outStr, ref config.fadeOutTime);
+
+            RenderSimpleValues.RenderPrefabInput("Prefab", ref config.satellite);
+            RenderSimpleValues.RenderFloatField("Size", ref config.satelliteSize);
+            RenderSimpleValues.RenderFloatField("Radius", ref config.radius);
+            RenderSimpleValues.RenderFloatField("y-offset", ref config.orbitYOffset);
+            RenderSimpleValues.RenderFloatFieldAsSlider("Initial Pos", ref config.initialAngle, 0, 360);
+            RenderSimpleValues.RenderFloatFieldAsSlider("Horizon Plane", ref config.horizonPlaneRotation, 0, 180);
+            RenderSimpleValues.RenderFloatFieldAsSlider("Inclination", ref config.inclination, 0, 180);
+            RenderSimpleValues.RenderFloatField("Speed", ref config.movementSpeed);
+            RenderSimpleValues.RenderEnumPopup<RotationType>("Rotation Type", ref config.satelliteRotation);
+
+            // If fixed rotation
+            if (config.satelliteRotation == RotationType.Fixed)
+            {
+                RenderSimpleValues.RenderVector3Field("Rotation", ref config.fixedRotation);
+            }
+            else if (config.satelliteRotation == RotationType.Rotate)
+            {
+                RenderSimpleValues.RenderVector3Field("Axis", ref config.rotateAroundAxis);
+                RenderSimpleValues.RenderFloatField("Rotation Speed", ref config.rotateSpeed);
```

Comment:
Several of these strings already exist at `unity-renderer/Assets/Rendering/ProceduralSkybox/ToolProceduralSkybox/Scripts/Editor/Procedural Skybox/SkyboxEditorLiterals.cs`, you could add the missing ones and reuse the ones that are already there